runscripts/pytest.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# Run pytest tests on this project.
+# Configured by `pytest.ini` and `.coveragerc`.
+
+pytest --cov=vivarium
```

Comment:
These runscripts are a great idea. Maybe we should also add a runscript for the docs tests? I think its just `doc/test.sh ci`. Is that right, @U8NWXD?

Reply:
Good idea. Done, and added Sphinx and sphinx-rtd-theme to `requirements.txt`.